### PR TITLE
Updated Croatian (hr) locale for v2

### PR DESCRIPTION
**Pull request by jerzabek**

### CODE REVIEW

**Comment by silvenon** [comment]
I'll take a look at this, don't merge yet. 👀

**Comment by kossnocorp** [comment]
@silvenon thanks! I'm going to ship it with 2.0.0-beta.4.

File: src/locale/hr/_lib/formatRelative/index.js [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
-  today: '[danas u] LT',
-  tomorrow: '[sutra u] LT',
-  nextWeek: 'dddd [u] LT',
-  other: 'L'
+  lastWeek: function(date) {
+    var day = date.getUTCDay()
+
+    switch (day) {
+      case 0:
+        return "'prošlu nedjelju u' p"
```
Comment (silvenon) [inline src/locale/hr/_lib/formatRelative/index.js]: I don't know if "prošle nedjelje" is more correct, but it's probably one of those expressions where both are equally correct. I'll ask around and amend later if I manage to find any resources about it.

**Comment by kossnocorp** [comment]
@silvenon thank you for the review. If you'll find something to improve please open a separate PR. I'm going to merge this one and publish a release.

**Pull request merged**